go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/cluster-node-tuning-operator
 
-go 1.21
+go 1.22.0
```

Comment:
`go: github.com/openshift/hypershift/api@v0.0.0-20240604072534-cd2d5291e2b7 requires go >= 1.22.0; switching to go1.22.3`

Reply:
There is a similar [PR for this](https://github.com/openshift/cluster-node-tuning-operator/pull/1068). Do we need 1.22.0 (vs. 1.22)?

Reply:
>  Do we need 1.22.0 (vs. 1.22)?
This is the output formatted and generated by the `go mod tidy`